internal/provider/resource_attest.go

```diff
@@ -143,7 +156,7 @@ func doAttest(ctx context.Context, data *AttestResourceModel, config AttestResou
 			SkipConfirmation: true,
 		},
 		RegistryOptions: options.RegistryOptions{
-			KubernetesKeychain: true,
+			RegistryClientOpts: config.popts.ropts,
```

Comment:
should this use `withContext`?

Reply:
Nah, the option factories that cosign uses pass in a context: https://github.com/sigstore/cosign/blob/14b76749318621205222f9ea02fbe541db599428/cmd/cosign/cli/options/registry.go#L68-L69C63